src/jquery.ime.inputmethods.js

```diff
@@ -198,6 +198,10 @@
 			name: 'फोनेटिक',
 			source: 'rules/hi/hi-phonetic.js'
 		},
+		'hoc': {
+			name: 'Ho transliteration',
+			source: 'rules/hoc-transliteration.js'
```

Comment:
This must be in rules/hoc/hoc-transliteration.js. Please create this directory and move the hoc-transliteration.js file into it.